procedures/foreman/branch.md.erb

```diff
@@ -56,19 +56,31 @@
   - [ ] Create [releases/<%= release %>/RPM-GPG-KEY-foreman](https://yum.theforeman.org/releases/<%= release %>/RPM-GPG-KEY-foreman) on yum.theforeman.org
   - [ ] Commit the [settings file](https://github.com/theforeman/theforeman-rel-eng/blob/master/releases/foreman/<%= release %>/settings) to the `theforeman-rel-eng` repository
 - [ ] Create new settings files for [client](https://github.com/theforeman/theforeman-rel-eng/blob/master/releases/client/<%= release %>/settings), ensure it has the rights `OSES` list and commit it too.
+- [ ] Create `configs/foreman/<%= release %>.yaml` based on the nightly and previous release. Ensure that any `based_on` entries are filled out and pointing to nightly version of tag.
+  - [ ] Generate mash configs
+    ```
+    bundle exec ./tools.rb mash-scripts configs/foreman/<%= release %>.yaml
+    sed -i '/strict_keys/ s/True/False/' mash_scripts/foreman/<%= release %>.0/foreman-plugins-<%= release %>-*.mash
+    ```
+  - [ ] Open PR with new config and mash scripts for review
+  - [ ] Create mash configuration on Koji, from tool_belt checkout:
+    ```
+    scp mash_scripts/foreman/<%= release %>.0/*.mash root@koji.katello.org:/etc/mash/
+    ```
+- [ ] Update mash script if needed - [collection-mash-split.py](https://github.com/theforeman/foreman-infra/blob/master/koji/collection-mash-split.py)
+- [ ] Add the new release to the JJB job definitions in [foreman-infra](https://github.com/theforeman/foreman-infra/tree/master/puppet/modules/jenkins_job_builder/files/theforeman.org/)
+  - [ ] Create a `<%= release %>.groovy` in `pipelines/vars/foreman/` describing the operating systems the release supports
+  - [ ] Add `<%= release %>` to the version list in `theforeman.org/yaml/includes/foreman_versions.yaml.inc`
+  - [ ] Add `<%= release %>` to the version list in `centos.org/jobs/foreman-pipelines.yml`
+  - [ ] Create `test_<%= release.tr('.', '_') %>_stable.yaml` and `test_proxy_<%= release.tr('.', '_') %>_stable.yaml` in `yaml/jobs/`
+- [ ] Open PR to remove any jobs that are related to end of life versions
+- [ ] Open PR to add <%= release %> to [Forklift versions config](https://github.com/theforeman/forklift/blob/master/vagrant/config/versions.yaml)
```

Comment:
So we noticed that merging this PR breaks the upgrade pipelines. Do we want to note anything about that it should only be merged after packaging branching has happened?

Reply:
How about this @ekohl ?
```suggestion
- [ ] Open PR to add <%= release %> to [Forklift versions config](https://github.com/theforeman/forklift/blob/master/vagrant/config/versions.yaml). Once the PR is merged, upgrade pipelines will fail, so do not merge it before packaging has been branched.
```

Reply:
Sounds good to me.